Hardcore.lua

```diff
@@ -540,6 +541,8 @@ function Hardcore:UNIT_SPELLCAST_SUCCEEDED(...)
 end
 
 function Hardcore:PLAYER_ENTERING_WORLD()
+	isPlayerDead = UnitIsDeadOrGhost("player")
```

Comment:
I think death triggering here was seen as a feature for verifiers. For example, if you disconnect death or alt for and die, then it will still trigger at least once upon logging back in.

Reply:
Didn't think about that. Removed it now.